Oasis CLI v0.8.0-gitc2cd406 has "panic: assignment to entry in nil map"

Oasis CLI [v0.8.0-gitc2cd406](https://github.com/oasisprotocol/cli/releases/tag/v0.8.0) shows next error:

```
oasis network list
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/oasisprotocol/cli/config.(*Config).migrateNetworks(0xc00001dc08?)
        github.com/oasisprotocol/cli/config/config.go:240 +0x459
github.com/oasisprotocol/cli/config.(*Config).Migrate(0x1c1b160)
        github.com/oasisprotocol/cli/config/config.go:179 +0x1c
github.com/oasisprotocol/cli/cmd.initConfig()
        github.com/oasisprotocol/cli/cmd/root.go:83 +0x34a
github.com/spf13/cobra.(*Command).preRun(...)
        github.com/spf13/cobra@v1.8.0/command.go:1017
github.com/spf13/cobra.(*Command).execute(0x1b7cc60, {0x1c7e060, 0x0, 0x0})
        github.com/spf13/cobra@v1.8.0/command.go:927 +0x5a2
github.com/spf13/cobra.(*Command).ExecuteC(0x1b78a40)
        github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/oasisprotocol/cli/cmd.Execute(...)
        github.com/oasisprotocol/cli/cmd/root.go:35
main.main()
        github.com/oasisprotocol/cli/main.go:6 +0x1b
```
v0.7.1-gitbcbf421 on the same system works fine.

--- a/config/config.go
+++ b/config/config.go
@@ -237,6 +237,9 @@ func (cfg *Config) migrateNetworks() bool {
 				continue
 			}
 
+			if net.ParaTimes.All == nil {
+				net.ParaTimes.All = make(map[string]*config.ParaTime)
+			}
 			net.ParaTimes.All[ptName] = pt
 			changes = true
 		}

